scripts/qemu/requirements.txt

```diff
@@ -0,0 +1 @@
+psutil
```

Comment:
please add an empty line at the end of this file

Reply:
Done